standard_test.go

```diff
@@ -2011,3 +2016,33 @@ func TestIsGameOver(t *testing.T) {
 		require.Equal(t, test.Expected, actual)
 	}
 }
+
+type JsonGameInput struct {
+	BoardState BoardState
+	Moves      []SnakeMove
+}
+
+func TestFromJson(t *testing.T) {
+	r := StandardRuleset{}
+
+	files, err := filepath.Glob("tests/standard/*.in.json")
+	require.NoError(t, err)
+
+	for _, file := range files {
```

Comment:
I decided to make a single 'Test` function, that looked for any number of files in the directory.

If it would be preferable to have a specific `Test` function for each file I can change it up! I thought this might be a slightly lower barrier to adding new test cases but 🤷